### ISSUE TITLE
obj_list (string array of the class labels) contains empty strings

### ISSUE THREAD

**Issue by pocaha** [zylo117/Yet-Another-EfficientDet-Pytorch#349]
Hey zylo117,

first and foremost, thank you for the great work. My question: the obj_list array in the coco.yml file (in projects folder) contains all the 80 classes of the coco dataset2017. However, there are also 11 empty strings. Don't they affect the model performance?

**Comment by pocaha** [comment]
@mnslarcher ?

**Comment by zylo117** [comment]
there are 90 classes in the pretrained weights.

**Comment by pocaha** [comment]
Coco2017 has 80 classes, ImageNet has way more than 90 classes, With which datasets were the weights pretrained? Also the number of the classes from the pretraining dataset don't have to match with the actual training set as else we would not use ImageNet pretrained weights for EfficientNet. Im training from scratch. By the way which pretraining do you mean? EfficientNet pretraining or coco pretraining?
